Added new gen-install-alpha command (#1673)
googleforgames/agones: site/content/en/docs/Installation/Install Agones/yaml.md

kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/install/yaml/install.yaml
```

{{% feature publishVersion="1.7.0" %}}
If you want to add additional parameters to `install.yaml` file, you can use helm directly to generate a custom file locally, but make sure new parameters correspond to the [following ones](https://agones.dev/site/docs/installation/install-agones/helm/#configuration).

> Example of adding feature-gates parameter:

Reviewer: ```suggestion
Example of setting Feature Gates and generateTLS parameters:
```

```

> helm3 pull --untar https://agones.dev/chart/stable/agones-{{< release-version >}}.tgz && \

Reviewer: ```suggestion
helm pull --untar https://agones.dev/chart/stable/agones-{{< release-version >}}.tgz && \
```
You may say in another place that this commands works only with Helm version 3.

kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/install/yaml/install.yaml
```

> {{% feature publishVersion="1.7.0" %}}

Reviewer 2: I don't think we need to feature block this - since we've already released all the helm3 features - it can go live straight away I think.

> If you want to add additional parameters to `install.yaml` file, you can use helm directly to generate a custom file locally, but make sure new parameters correspond to the [following ones](https://agones.dev/site/docs/installation/install-agones/helm/#configuration).

Reviewer 2: ```suggestion
If you want to change the parameters in the `install.yaml` file, you can use helm directly to generate a custom file locally, but make sure new parameters correspond to the [following ones](https://agones.dev/site/docs/installation/install-agones/helm/#configuration).
```
?

Author: done
